platform/prototype/README.md

```diff
@@ -264,6 +264,61 @@ This gives you the ultimate freedom, as your apps doesn't need to know where the
 
 Finally, you can change the application source code using tools like GitPod. Check the applicaiton repository [Prototype Apps](https://github.com/salaboy/prototype-apps). If you install the [GitPod Plugin on Chrome](https://chrome.google.com/webstore/detail/gitpod-always-ready-to-co/dodmmooeoklaejobgleioelladacbeki) you can open the repository on a remote environment that has all the tools ready to work with the application and deploy it to Kubernetes Clusters. 
 
+### Internal Developer Portal
+
+In order to keep track of all of the new services, applications, documentation, resources, infrastructure, etc. that your teams will continue to create throughout the lifecycles of your intiatives - having an internal developer portal may be a very valuable addition to your platform.
+
+First we'll build the backstage image locally and make it available for the cluster
+
+```
+yarn --cwd backstage build:backend
```

Comment:
@salaboy if we use a public image (like [this](https://roadie.io/backstage/docker-image/) one), we cannot configure things like the Kubernetes integration or add any plugins ourselves.

This is one of the downsides of backstage, the flexibility can be great so we can easily add plugins however it's very difficult to build/maintain unless you use a SAAS provider like Roadie who does the abstraction for you.

Reply:
If you check Backstage's documentation, the best practices are displayed here: https://backstage.io/docs/deployment/k8s

I've abstracted many of the steps in the documentation by configuring the backstage directory to have most of the configuration set. 

A different step we could take if we really don't want the user to need to run the yarn command is we could build a public image ourselves. The user would be unable to make any changes to the backend which would greatly reduce the flexibility of Backstage, but if we're okay with that then it could work

Reply:
@asarenkansah thanks for the pointers.. I was expecting an answer like this.. so I need to try it out and make a decision there.. I think that for this work and this example.. I will create a separate repo to contain our custom backstage configuration.. instead of pushing it here.. do you mind if I create a repo here and give you access to push?